app/src/main/res/drawable/search_view_background.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would name this xml like bg_search_view
Example:
bg - background
ic - icon

Reply:
improve